src/views/Home/components/About/AboutBox/AboutBox.scss

```diff
@@ -0,0 +1,47 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.about-box {
+    z-index: 1;
+    border-radius: 33px;
+    width: 80%;
+    min-height: 331px;
+    height: 100%;
+    background-color: var(--color-dark-navy-blue);
+    margin: 50px 0 50px 0;
+    &__title {
+        background-color: var(--color-text-primary);
+        color: var(--color-dark-navy-blue);
+        font-family: Source Sans Pro, "sans-serif";
+        font-style: normal;
+        font-weight: bold;
+        line-height: 75px;
+        padding: 0 2% 0 2%;
+        border-top-left-radius: 33px;
+        border-top-right-radius: 33px;
+        @include responsiveness.font($responsive: 4.1vw, $min: 35px, $max: 80px);
+        @include responsiveness.screen(smallest, phone) {
+            @include responsiveness.font($responsive: 7.1vw, $min: 22px, $max: 40px);
+            padding: 0 26px 0 30px;
+        }
+    }
+    &__description {
+        font-family: Lato, "sans-serif";
+        line-height: 42px;
```

Comment:
I don't think we need to set `line-height` actually, the default looks fine to me. The same goes for line 40.

Reply:
Oh yeah this seems a lot more natural than the default figma value for a responsiveness.